src/pages/Home/components/Editor/components/InputTextarea/InputTextArea.test.js

```diff
@@ -0,0 +1,21 @@
+import { getNewGuitarBlock } from "../../utils/EditorUtils";
+import InputTextarea from "./InputTextarea";
```

Comment:
seems this is throwing the error, prob just need to adjust the function to be uniform in casing?

Reply:
Fosho. Good catch. It's fixed locally but github doesn't want to take it. Had to change it manually on GH